tests/test_git_scraper.py

```diff
@@ -57,6 +60,12 @@ def get_repo(repo_name):
     directory = os.path.join(test_dir, repo_name)
     repo = Repo.init(directory)
 
+    # We need to synthesize the time stamps of commits to each be a second
+    # apart, otherwise the commits may be at exactly the same second, which
+    # means they won't always sort in order, and this the merging of identical
+    # metrics in adjacent commits may not happen correctly.
+    base_time = time.time()
```

Comment:
@fbertsch: This was a fun one!  I think it's only a bug in the way things are tested, though, not in probe scraper itself.

Reply:
Yes, I agree! Good catch on that one - much more performant than a `sleep` command at the end of each iteration :)